parser/misc.go

```diff
@@ -686,6 +698,9 @@ var tokenMap = map[string]int{
 	"SQL_TSI_WEEK":             sqlTsiWeek,
 	"SQL_TSI_YEAR":             sqlTsiYear,
 	"SQL":                      sql,
+	"SQLWARNING":               sqlwarning,
+	"SQLEXCEPTION":             sqlexeption,
+	"SQLSTATE":                 sqlstate,
```

Comment:
```suggestion
	"SQLEXCEPTION":             sqlexeption,
	"SQLSTATE":                 sqlstate,
	"SQLWARNING":               sqlwarning,
```
maybe It's better to keep all these new added token in alphabet order